src/Microsoft.Health.Dicom.Core/Messages/ExtendedQueryTag/GetExtendedQueryTagErrorsResponse.cs

```diff
@@ -0,0 +1,24 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using EnsureThat;
+using Microsoft.Health.Dicom.Core.Features.ExtendedQueryTag;
+
+namespace Microsoft.Health.Dicom.Core.Messages.ExtendedQueryTag
+{
+    public class GetExtendedQueryTagErrorsResponse
+    {
+        public GetExtendedQueryTagErrorsResponse(IReadOnlyCollection<ExtendedQueryTagError> extendedQueryTagErrors)
+        {
+            ExtendedQueryTagErrors = EnsureArg.IsNotNull(extendedQueryTagErrors);
+            ErrorCount = extendedQueryTagErrors.Count;
+        }
+
+        public int ErrorCount { get; }
```

Comment:
Do we need an explicit `Count` if we providing the elements as well.
- In JSON this is an array which has a length when deserialized
- In .NET, this is a collection which has a `Count`

What does this mean when we're paginated? It looks like it is the amount in the page at the moment

Reply:
will remove and add in the pagination PR